src/app/(auth)/(navigationsBarLayout)/@chatDialog/page.tsx

```
@@ -2,15 +2,17 @@
 import { ChatRoom, ChatRoomList } from '@/features/chat';
 import { DialogTitle } from '@radix-ui/react-dialog';
-import { use } from 'react';
+import { useSearchParams } from 'next/navigation';
 
-interface ChatDialogProps {
-  searchParams: Promise<{ 'room-id': string; title: string }>;
-}
-export default function ChatDialog({ searchParams }: ChatDialogProps) {
-  const roomId = use(searchParams)['room-id'];
-  const title = use(searchParams)['title'];
+export default function ChatDialog({}) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)